changelog.md

```diff
@@ -2,7 +2,9 @@
 
 ## UNRELEASED
 
-_Nothing yet..._
+**Fixed:**
```

Comment:
It means that if this change might break consumers build / design then it's a major bump.

However that being said .. if consumers are using `innerPadding` and it has a similar visual effect then it's not a breaking change.